src/test/java/de/plushnikov/intellij/plugin/inspection/BuilderInspectionTest.java

```diff
@@ -34,4 +34,8 @@ public void testBuilderDefaultsWarnings() throws Exception {
     //TODO implement test after adding support for Builder.Default
```

Comment:
I assume this todo is to be removed now? If not, i think we are missing part of the functionality.

Reply:
I saw that one, but I am not sure what to do. It seems like it is an unfinished test that should test for a warning, but the default functionality works now, so a warning seems not necessary.

I think it can be removed. If you agree, I will commit a removal.

Reply:
I would say simply drop the TODO and then we can merge this